resources/asciidoctor/.rubocop.yml

```diff
@@ -0,0 +1,116 @@
+AllCops:
+  TargetRubyVersion: 2.3
+
+Layout/AlignHash:
+  Enabled: false
+
```

Comment:
Are you intending to turn up more checks over time? How many does this leave enabled? :laughing:

Reply:
There are at least three!